grammar_codegen/testdata/expected_json_grammar.h

```diff
@@ -102,48 +113,64 @@ class ArraySubNode4;
 class ElementsSubNode5;
 class ElementsSubNode6;
 class STRINGSubNode7;
-class NUMBERSubNode8;
-class NUMBERSubNode9;
-class INTEGERSubNode10;
-class INTEGERSubNode11;
+class STRINGSubNode9;
+class STRINGSubNode8;
+class NUMBERSubNode10;
+class NUMBERSubNode11;
 class INTEGERSubNode12;
-class DIGITSSubNode13;
-class EXPONENTSubNode14;
-class EXPONENTSubNode15;
-class WSPACESubNode16;
-class Literal7;
-class Literal11;
-class Literal8;
+class INTEGERSubNode13;
+class INTEGERSubNode14;
+class DIGITSSubNode15;
+class EXPONENTSubNode16;
+class EXPONENTSubNode17;
+class WSPACESubNode18;
+class Literal14;
+class Literal18;
+class Literal15;
+class Literal13;
 class Literal6;
-class Literal5;
-class Literal3;
 class Literal12;
+class Literal3;
+class Literal19;
 class Literal4;
-class Literal13;
+class Literal5;
+class Literal20;
+class Literal7;
+class Literal8;
 class Literal1;
-class Literal2;
-class Literal0;
 class Literal9;
+class Literal2;
 class Literal10;
+class Literal11;
+class Literal0;
+class Literal16;
+class Literal17;
 class CharSet3;
 class CharSet1;
 class CharSet2;
 class CharSet0;
 
-inline constexpr absl::string_view kStrLiteral7 = "+";
-inline constexpr absl::string_view kStrLiteral11 = ",";
-inline constexpr absl::string_view kStrLiteral8 = "-";
-inline constexpr absl::string_view kStrLiteral6 = ".";
-inline constexpr absl::string_view kStrLiteral5 = "0";
+inline constexpr absl::string_view kStrLiteral14 = "+";
+inline constexpr absl::string_view kStrLiteral18 = ",";
+inline constexpr absl::string_view kStrLiteral15 = "-";
+inline constexpr absl::string_view kStrLiteral13 = ".";
+inline constexpr absl::string_view kStrLiteral6 = "/";
+inline constexpr absl::string_view kStrLiteral12 = "0";
 inline constexpr absl::string_view kStrLiteral3 = ":";
-inline constexpr absl::string_view kStrLiteral12 = "[";
+inline constexpr absl::string_view kStrLiteral19 = "[";
 inline constexpr absl::string_view kStrLiteral4 = "\"";
-inline constexpr absl::string_view kStrLiteral13 = "]";
+inline constexpr absl::string_view kStrLiteral5 = "\\";
```

Comment:
Without the fix in antlr_frontend.cc, this (string of length one) `"\\"` would be (a string of length two) `"\\\\"`.